### PR TITLE
fix #3955 feat(nimbus): add monitoring dashboard url to experiment query

### LINKED ISSUE #3955: Add monitoring dashboard URL to NimbusExperiment db model
The `rapid` project hits `/api/v3/experiments/${experimentSlug}/` and receives a `monitoring_dashboard_url` string back.

Add this property to NimbusExperiment so we can display it in the client through the `experimentBySlug` query.

┆Issue is synchronized with this [Jira Task](https://jira.mozilla.com/browse/EXP-571)
┆Issue Number: EXP-571

### PR DESCRIPTION
**Pull request by jaredlockhart**
Because

* Until we have the visualization doing monitoring we'll need to link out to grafana

This commit

* Adds a grafana monitoring link to the experiment graphql query

### CODE REVIEW

**Comment by LZoog** [comment]
@jaredlockhart I had closed this issue, see comment: https://github.com/mozilla/experimenter/issues/3955#issuecomment-729215308

Though with you already doing the work here, I think it's still fine to merge if you want to reopen the issue before merging this. It'll make EXP-607 (adding the URL) that much easier 🤷‍♀️

**Comment by bbangert** [comment]
@jaredlockhart do you still want a review on this?

**Comment by jaredlockhart** [comment]
@LZoog Yeah it's just easy to do both since this code is already written, only took a second, I'll add the dates as well.

@bbangert yes please.

**Review by bbangert — APPROVED** [review]
LGTM!

**Pull request merged**
